axlearn/common/utils.py

```diff
@@ -1459,3 +1459,30 @@ def validate_contains_paths(x: Nested[Tensor], paths: Sequence[str]):
                 f"Input is expected to contain '{path}'; "
                 f"instead, it contains: '{jax.tree_structure(x)}'."
             ) from e
+
+
+def save_only_these_regex_patterns(*regex_patterns_to_save):
```

Comment:
Add an annotation for the args and return type?

Reply:
Removed this function from the PR